packages/icons/src/metadata.ts

```diff
@@ -0,0 +1,1816 @@
+  },
+  "user-circle-minus": {
+    category: "People & User",
+  },
+  "user-clock": {
+    category: "People & User",
+  },
+  "user-company-id": {
+    category: "People & User",
+  },
+  "user-edit": {
+    category: "People & User",
+  },
+  "user-group": {
+    category: "People & User",
+  },
+  "user-id": {
+    category: "People & User",
+  },
+  "user-key": {
+    category: "People & User",
+  },
+  "user-location-pin": {
+    category: "People & User",
+  },
+  "user-lock": {
+    category: "People & User",
+  },
+  "user-message-minus": {
+    category: "People & User",
+  },
+  "user-message-plus": {
+    category: "People & User",
+  },
+  "user-minus": {
+    category: "People & User",
+  },
+  "user-nametag": {
+    category: "People & User",
+  },
+  "user-plus": {
+    category: "People & User",
+  },
+  "user-settings": {
+    category: "People & User",
+  },
+  "user-shield": {
+    category: "People & User",
+  },
+  "user-suit-nametag": {
+    category: "People & User",
+  },
+  "user-x": {
+    category: "People & User",
+  },
+  vibration: {
+    category: "Communication",
+  },
+  videocam: {
+    category: "Devices & Electronics",
+  },
+  "videocam-disabled": {
+    category: "Devices & Electronics",
+  },
+  virus: {
+    category: "Health, Medicine & Hygiene",
+  },
+  "virus-shield": {
+    category: "Health, Medicine & Hygiene",
+  },
+  "volume-disabled": {
+    category: "Action & Interface",
+  },
+  "volume-down": {
+    category: "Action & Interface",
+  },
+  "volume-mute": {
+    category: "Action & Interface",
+  },
+  "volume-up": {
+    category: "Action & Interface",
+  },
+  voucher: {
+    category: "Commerce & Shopping",
+  },
+  walking: {
+    category: "Transport & Travel",
+  },
+  wallet: {
+    category: "Commerce & Shopping",
+  },
+  warehouse: {
+    category: "Commerce & Shopping",
+  },
+  webhook: {
+    category: "Software & Services",
+  },
+  windmill: {
+    category: "Weather & Climate",
+  },
+  windows: {
+    category: "Software & Services",
+  },
+  wlan: {
+    category: "Connectivity",
+  },
+  wrench: {
+    category: "Objects",
+  },
+  x: {
+    category: "Essentials",
+  },
+  "x-small": {
+    category: "Essentials",
+  },
+  "zoom-in": {
+    category: "Action & Interface",
+  },
+  "zoom-out": {
+    category: "Action & Interface",
+  },
+} as const satisfies Record<string, IconMetadata>;
+
+/**
+ * Grouped metadata of all available icons by category.
+ */
+export const ICON_CATEGORIES = groupIconsByCategory();
+
+export type IconMetadata = {
+  category: string;
+  deprecated?: boolean;
+  aliases?: string[];
+};
+
+export type GroupedIconCategory = {
+  iconName: keyof typeof ICON_METADATA;
+  metadata: IconMetadata;
+};
+
+export type IconCategories = Record<string, GroupedIconCategory[]>;
+
+/**
+ * Groups all available icon metadata by category.
+ * Categories and icons will be sorted alphabetically.
+ */
+function groupIconsByCategory() {
```

Comment:
don't we usually use arrow functions? is there a need for the named function here?

```suggestion
const groupIconsByCategory = () => {
```

Reply:
only thing you need to change then is to put it above ICON_CATEGORIES so you don't get a warning "used before being assigned